__tests__/__mocks__/request.js

```diff
@@ -55,14 +74,21 @@ const httpMock = {
     // TODO better way to do this
     const httpModule = options.uri.href.startsWith('https:') ? https : http;
 
+    const {token, registry} = getAuthInfo(options.uri.path) || {};
```

Comment:
PR looks good.
My only concern is that this `/fixtures/authed-packages/config` is used for all the unit tests that make requests.
Can you set/reset this fixture only for the test you need?